pallets/automation-time/src/tests.rs

```diff
@@ -327,32 +365,35 @@ fn schedule_transfer_with_dynamic_dispatch() {
 		let recipient = AccountId32::new(BOB);
 		assert_eq!(Balances::free_balance(recipient.clone()), 127);
 
-		assert_eq!(
+		let mut condition: BTreeMap<String, String> = BTreeMap::new();
+		condition.insert(String::from("type"), String::from("time"));
+		condition.insert(String::from("timestamp"), format!("{}", SCHEDULED_TIME));
+
+		assert!(contains_events(
 			my_events,
-			[
-				RuntimeEvent::System(frame_system::Event::NewAccount {
-					account: recipient.clone()
-				}),
-				RuntimeEvent::Balances(pallet_balances::pallet::Event::Endowed {
-					account: recipient.clone(),
-					free_balance: 127,
+			vec![
+				RuntimeEvent::AutomationTime(crate::Event::TaskTriggered {
+					who: account_id.clone(),
+					task_id: task_id.clone(),
+					condition,
+					encoded_call: call.encode(),
+					abort_errors: vec![],
 				}),
 				RuntimeEvent::Balances(pallet_balances::pallet::Event::Transfer {
 					from: account_id.clone(),
 					to: recipient.clone(),
 					amount: 127,
 				}),
-				RuntimeEvent::AutomationTime(crate::Event::DynamicDispatchResult {
+				RuntimeEvent::AutomationTime(crate::Event::TaskExecuted {
 					who: account_id.clone(),
 					task_id: task_id.clone(),
-					result: Ok(()),
 				}),
 				RuntimeEvent::AutomationTime(crate::Event::TaskCompleted {
 					who: account_id,
 					task_id,
 				}),
 			]
-		);
+		));
```

Comment:
@v9n, this list is the expected event firing behavior for successful task executions. The PR is ready for review. Could you double check it, especially the dev code in automation-time/src/lib.rs?